### PR TITLE
[SDP-268] Add organization creation from the app

### PR DESCRIPTION
**Pull request by freakinhuge**
Add organization creation from the app.

### CODE REVIEW

File: app/controllers/admin/organizations_controller.rb [hunk 1]
```diff
@@ -57,13 +58,13 @@ def edit
   # POST /organizations.json
   def create
     @organization = Organization.new(organization_params)
-    authorize @organization
```
Comment (t2) [inline app/controllers/admin/organizations_controller.rb]: Did you mean to remove this validation?

Comment (freakinhuge) [reply app/controllers/admin/organizations_controller.rb]: We were originally going to allow non-admin create of organizations, but moved it all over to the home controller. I'll put that back.

**Comment by cjarrett** [comment]
@freakinhuge could you resolve this conflict and push back up, please?

**Comment by cjarrett** [comment]
@freakinhuge we should move the update_auth0_callback_urls and assign_dns method to after the organization is approved. also, should send a tokenized invite a la 287.

**Comment by freakinhuge** [comment]
Are we sending the invite when the organization is approved?

**Pull request merged**